src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/UnscopedRefAttribute.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    /// <summary>
+    /// Used to indicate a byref escapes and is not scoped.
+    /// </summary>
+    /// <remarks>
+    /// There are several cases where the C# compiler treats a <c>ref</c> as implicitly
+    /// <c>scoped</c> - where the compiler does not allow the <c>ref</c> to escape the method.
+    ///
+    /// For example:
+    /// 1. <c>this</c> for struct instance methods
+    /// 2. <c>ref</c> parameters that refer to <c>ref struct</c> types
+    /// 3. <c>out</c> parameters
+    ///
+    /// This attribute is used in those instances where the <c>ref</c> should be allowed to escape.
```

Comment:
Not sure how overboard the runtime likes to go with XML docs, but this could use some more specific nodes?
For instance:
```suggestion
    /// <para>
    /// There are several cases where the C# compiler treats a <see langword="ref"/> as implicitly
    /// <see langword="scoped"/> - where the compiler does not allow the <see langword="ref"/> to escape the method.
    /// </para>
    /// <para>
    /// For example:
    /// <list type="number">
    ///     <item><see langword="this"/> for <see langword="struct"/> instance methods.</item>
    ///     <item><see langword="ref"/>parameters that refer to <see langword="ref"/> <see langword="struct"/> types.</item>
    ///     <item><see langword="out"/> parameters.</item>
    /// </list>
    /// </para>
    /// <para>
    /// This attribute is used in those instances where the <see langword="ref"/> should be allowed to escape.
    /// </para>
```

The results is going from this:

![](https://user-images.githubusercontent.com/10199417/180304692-1bebc7b4-0cdd-4e2f-8e39-0742b9a95f69.png)

To this:

![](https://user-images.githubusercontent.com/10199417/180305098-e88267bb-cc9a-488b-9d5c-92fb04a81776.png)

Much better formatted, and with syntax highlight as well 😄